test/development/app-dir/dynamic-in-generate-params/index.test.ts

```diff
@@ -0,0 +1,43 @@
+import { type NextInstance, createNextDescribe } from 'e2e-utils'
+
+async function getLastModifiedTime(next: NextInstance, pathname: string) {
+  const content = await (await next.fetch(pathname)).text()
+  return content.match(/<lastmod>([^<]+)<\/lastmod>/)[1]
+}
+
+createNextDescribe(
+  'app-dir - dynamic in generate params',
```

Comment:
Why do we want this as specifically `development` test, would it be good to have as a normal `e2e` test so we ensure both dev and start are working as expected?

Reply:
def an accident 😂 was intended to put them in e2e..